tests/test_jwt_utils.py

```diff
@@ -30,19 +31,24 @@ def test_get_matching_missing_key(self) -> None:
 
 
 class TestDecodeJWT:
-    @patch("lbz.jwt_utils.get_matching_jwk", return_value={})
-    def test_did_not_find_matching_jwk(
-        self, get_matching_jwk_mock: MagicMock, caplog: pytest.LogCaptureFixture
-    ) -> None:
-        with pytest.raises(Unauthorized):
-            decode_jwt("x")
-        get_matching_jwk_mock.assert_called_once_with("x")
-        assert "Failed decoding JWT with following details" in caplog.text
+    def test_did_not_find_matching_jwk(self, caplog: pytest.LogCaptureFixture) -> None:
+        public_key = deepcopy(SAMPLE_PUBLIC_KEY)
+        public_key["kid"] = "9494ad75-aaaa-aaaa-aaaa-c1a17da22b35"
+        token_payload = {
+            "iss": "test-issuer",
+            "aud": "test-audience",
+            "kid": "x",
+        }
+        jwt_token = encode_jwt(token_payload, SAMPLE_PRIVATE_KEY)
```

Comment:
According to the above comments:
```suggestion
        jwt_token = encode_jwt(data={}, private_key_jwk=private_key)
```